src/components/forms/TextInput.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { throttle, debounce } from 'throttle-debounce';
+import IconInput from 'components/forms/IconInput';
+import DecoratedInput from 'components/forms/DecoratedInput';
+
+export const throttleValue = 500;
+
+const TextInput = ({
+  label,
+  errorText,
+  throttleOnChange,
```

Comment:
@akdetrick how do you feel about `onThrottleChange`?

Reply:
`onThrottledChange` makes the most sense to me (past tense)

Reply:
`onThrottledChange` (past tense Throttled) makes the most sense to me